src/main/java/dk/cngroup/lentils/controller/ClientContactController.java

```diff
@@ -0,0 +1,50 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.security.CustomUserDetails;
+import dk.cngroup.lentils.service.ContactService;
+import dk.cngroup.lentils.service.CypherService;
+import dk.cngroup.lentils.service.ScoreService;
+import dk.cngroup.lentils.service.StatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/contact")
+public class ClientContactController {
+    private static final String VIEW_CONTACT_LIST = "clientcontact/list";
```

Comment:
To view by mohlo byt spise `client/contact.html`

Reply:
opraveno